javascript/javascript-core/intro-js/javascript-basics-part-1.md

```diff
@@ -56,19 +49,26 @@ firstName = "Changed"; // this works
 
 We'll learn more about the differences between `let`, `const`, and `var` later on.
 
-### Semicolon
 
-Statements in JavaScript end in a `;`. Based on the ECMAScript[1] rules however, adding a `;` isn't actually required (although it is common practice):
+### Case Sensitivity
 
-```javascript
-let firstName = "Enki";
-//                    ^
+Everything in JavaScript is case sensitive.
+
+```js
+let password;
+// Is not the same as
+let pasSword;
+// or
+let PASSWORD;
 ```
 
-Omitting a semicolon in certain situations can cause problems. JavaScript has a feature called Automatic Semicolon Insertion (ASI) which means that, if you omit a semicolon, JavaScript will automatically add it where it thinks it should go. 
 
-This can sometimes lead to unexpected results[2]. Thus, it's usually recommended to insert the `;` yourself, or use a tool such as [Prettier](https://prettier.io) to do it for you.
+### Keywords
+
+Like most languages, there are a number of reserved words in JavaScript that you cannot use to name your functions and variables (e.g. `var`, `let`, `new`).
+
 
+> 💡 If you're unsure what a variable is, check out the [variables lesson](https://app.enki.com/insights/programs-and-data) in our [Coding Intro](https://app.enki.com/skill/coding-intro).
```

Comment:
let's add "topic" here because otherwise the phrase reads "... in our Coding Intro"
```suggestion
> 💡 If you're unsure what a variable is, check out the [variables lesson](https://app.enki.com/insights/programs-and-data) in our [Coding Intro](https://app.enki.com/skill/coding-intro) topic.
```